src/posts/lucky_0.23.cr

```diff
@@ -0,0 +1,219 @@
+
+    ```crystal
+    # run the query once, and always return that value
+    user(4)
+
+    # always run the query and return the value
+    user__uncached(4)
+    ```
+
+    ### Security and more
+    
+    Some of the HTML page helpers like `highlight`, and `simple_format` would allow for unescaped HTML.
+    This could potentially be a security issue, so that's been fixed. All of these methods will escape the
+    HTML by default now. If you need the unescaped version, you can pass `escape: false`.
+
+    ```crystal
+    highlight("<p>This is a beautiful morning, but also a beautiful day</p>", "beautiful", escape: false)
+    ```
+
+    We've also fixed a few issues related to cookies.
+    * You can now set a `samesite` default.
+    * Exceptions aren't raised when trying to delete a cookie that doesn't exist
+    * Cookies are written at the correct time
+
+    ## Routing Updates
+
+    We have added a new `route_prefix` macro that alllows you to prefix your routes with
+    some path. This really helps with APIs where you may want to prefix all your routes with
+    something like `/api/v1`.
+
+    ```crystal
+    abstract class ApiAction < Lucky::Action
+      accepted_formats [:json], default: :json
+
+      route_prefix "/api/v1"
+    end
+
+    class Posts::Index < ApiAction
+      # GET /api/v1/posts`
+      get "/posts" do
+        #...
+      end
+    end
+    ```
+
+    Sometimes you may have a route with optional path params. For example, imagine a list of
+    blog posts that can be optionally filtered by appending the year, or year and month.
+
+    ```crystal
+    class Posts::Index < BrowserAction
+      # Matches GET /posts
+      # GET /posts/2020
+      # GET /posts/2020/07
+      get "/posts/?:year/?:month" do
+        # ...
+      end
+    end
+    ```
+
+    On top of all the neat routing features, we were also able to squeeze a little more performance
+    out thanks to [@matthewmcgarvey](https://github.com/luckyframework/lucky_router/pull/26)!
+
+    ## CLI Task Args
+
+    Writing CLI tasks for your application are pretty common, but they can also be a bit complicated.
+    Lucky already comes with a ton of tasks like `lucky db.migrate`, and `lucky gen.page About`. When
+    you start to write your own, you may need to pass in arguments from the CLI. We break these in to 3 different type.
+
+    * `arg` - This is the most standard CLI argument type. e.g. `-m User` or `--model=User`.
+    * `switch` - Just like `arg`, but with out a value. Returns `true` if this flag is passed. e.g. `-v`, `-h`
+    * `positional_arg` - When you want to pass args that are not flags. Lucky uses this with generators like
+      `lucky gen.page About` where `About` is a positional arg.
+
+    ```crystal
+    # lucky search.reindex User
+    class Search::Reindex < LuckyCli::Task
+      summary "Reindex records for a model"
+    
+      positional_arg :model, "Specify which model to reindex", required: true
```

Comment:
I thought we changed this to be require by default? Maybe I'm misremembering, but might be worth checking

Reply:
Good catch! We did, I just never updated my example I was looking at to remember how this worked 😂